packages/page-council/src/Overview/Voters.tsx

```diff
@@ -28,19 +28,30 @@ function Voters ({ balance, voters }: Props): React.ReactElement<Props> {
 
   return (
     <>
-      <td className='expand'>
-        <ExpanderScroll
-          className={balance ? '' : '--tmp'}
-          renderChildren={renderVoters}
-          summary={<FormatBalance value={balance} />}
-        />
-      </td>
-      <td className='number'>
-        {voters
-          ? formatNumber(voters.length)
-          : <span className='--tmp'>123</span>
-        }
-      </td>
+      {voters ? (
+        <>
+          <td className='expand'>
+            <ExpanderScroll
+              className={balance ? '' : '--tmp'}
+              renderChildren={renderVoters}
+              summary={<FormatBalance value={balance} />}
+            />
+          </td>
+          <td className='number'>
+            {voters
+              ? formatNumber(voters.length)
+              : <span className='--tmp'>123</span>
+            }
+          </td>
+        </>
+      ) : (
+        <>
+          {/* empty <td>'s necessary to maintain row sizes */}
+          <td></td>
+          <td></td>
+        </>
+      )}
+
```

Comment:
```suggestion
      {voters
        ? (
          <>
            <td className='expand'>
              <ExpanderScroll
                className={balance ? '' : '--tmp'}
                renderChildren={renderVoters}
                summary={<FormatBalance value={balance} />}
              />
            </td>
            <td className='number'>
              {voters
                ? formatNumber(voters.length)
                : <span className='--tmp'>123</span>
              }
            </td>
          </>
        )
        : <td colSpan={2} />
      )}
```

Slight rework to address linting issue.